mailpoet/tests/acceptance/Subscribers/ManageSubscribersCest.php

```diff
@@ -70,6 +72,21 @@ public function viewSubscriberList(\AcceptanceTester $i) {
     $i->searchFor('Alec Saunders');
     $i->waitForText('Alec Saunders');
     $i->seeNoJSErrors();
+
+    $i->wantTo('see subscribed date changed upon switching timezone');
+    $subscriberFirstName = 'John ';
```

Comment:
Thanks for review @lysyjan 
I updated the test as suggested. First I wanted to make like in JS `'${var} ${var}'` but the wanted to avoid doing like this  `$i->waitForText($subscriberFirstName . ' ' . $subscriberLastName);` and added that simple space in the name which worked fine. Now it's like suggested.